labrad/protocol.py

```diff
@@ -333,5 +373,92 @@ def __init__(self, source, context, target):
     def __repr__(self):
         return 'MessageContext(source=%s, ID=%s, target=%s)' % (self.source, self.ID, self.target)
 
-# factory for churning out LabRAD connections
+
+# factory for creating LabRAD connections
 factory = protocol.ClientCreator(reactor, LabradProtocol)
```

Comment:
Renamed to `_factory`. The point of this change is that no one should ever have to touch things like twisted ClientFactory instance; just call the `connect` function. Since this is only used in two places about 20 lines down in this one file, I think that name and the comment is plenty clear.